.gitignore

```diff
@@ -57,4 +57,7 @@ bin/
 [Bb][Ii][Nn]/
 
 # ignore googletest items
-test/googletest-*/
\ No newline at end of file
+test/googletest-*/
+
+#vscode defaults
+.vscode
```

Comment:
This should likely be `.vscode/` and then you may need to untrack all of the files in the folder